src/spendings/spendings-list/spendings-list.tsx

```diff
@@ -0,0 +1,21 @@
+import { ISpendings } from '../../api/types/ISpendings'
+import { SpendingCard } from '../spendings-card/spendings-card'
+
+export function SpendingList({ items }: { items: ISpendings[] }) {
```

Comment:
сделать через 
`export interface SpendingsListProps {`
`items?: SpendingsCardProps[]`

```suggestion
export function SpendingsList({ items }: SpendingsListProps) {
```